sycl/test/basic_tests/stream/stream.cpp

```diff
@@ -1,7 +1,3 @@
-// TODO: Enable compilation w/o -fno-sycl-early-optimizations option.
```

Comment:
Looks like tests are still failing

Reply:
Yeah, it was passing with a previous patch set. Converting PR back to draft for now.